.github/workflows/config/spelling_allowlist.txt

```diff
@@ -214,19 +224,27 @@ lossy
 lvalue
 makefiles
 merchantability
+mgpu
 multithreaded
 mutex
 namespace
 namespaces
 natively
 normalization
+np
 nullary
+nvidia
+occa
+occb
+openfermion
```

Comment:
These all look like code items and probably shouldn't be added to the spell check allow list.